types/order_items/validator.js

```diff
@@ -0,0 +1,18 @@
+require('json5/lib/register');
+const { Validator } = require('jsonschema');
+
+const schema = require('./json-schema.json');
+
+const v = new Validator();
+
+/**
+ * Check if passed object is valid or not
+ * @param obj {object}
+ * @returns {boolean}
+ */
+module.exports = obj => {
+  if (process.env.DEBUG) {
+    console.log('Errors:', v.validate(obj, schema).errors);
+  }
```

Comment:
Do we need this block of code after merging into master or it's here because of your development?

Reply:
I suppose - it's better to keep. 

I'm going move this code to common files (not related to certan type )